test/integration_test/specs/kubevirt-fedora/fedora-vm.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  #  annotations:
+  #    kubevirt.io/latest-observed-api-version: v1
+  #    kubevirt.io/storage-observed-api-version: v1alpha3
+  labels:
+    app: fedora-vm
+    vm.kubevirt.io/template: fedora-server-small
+    vm.kubevirt.io/template.namespace: openshift
```

Comment:
Just curious. Are all these labels required or will they be added automatically by kubevirt?

Reply:
I think we add them, either via the UI or specs.